src/main/config/codelist/local/thesauri/theme/EC_Content_Scope.rdf

```diff
@@ -2,7 +2,14 @@
 <rdf:RDF
 	xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
 	xmlns:ns2="http://www.w3.org/2004/02/skos/core#"
-	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+	xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+	xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+	xmlns:dc="http://purl.org/dc/elements/1.1/">
+
+  <skos:ConceptScheme rdf:about="http://geonetwork-opensource.org/thesaurus/naturalearth-and-seavox">
+    <dc:title>EC Content Scope</dc:title>
```

Comment:
All others have ECCC but this one only has EC? Is that on purpose?

Reply:
I expect I will be updating all of the titles with more "official" versions.
I will fix this...